lib/sms/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig SMS
+	bool "SMS subscriber"
+	select AT_CMD
+	select AT_CMD_PARSER
+	select AT_NOTIF
```

Comment:
We need some kind of dependency, e.g. on bsd lib or nrf91 to hide this from nrf52 platforms.

Reply:
`	depends on TRUSTED_EXECUTION_NONSECURE
`

is what bsd library uses and is as good as any I suppose.

Reply:
Ah, no, then it will be visible on nrf53 ...

Reply:
Actually, the KConfig dependencies for `bsdlib` are wrong.
Using `SOC_SERIES_NRF91X` has a dependency to `BOARD_NRF9160_PCA10090` as well (secure).
I do not think I can use `BOARD_NRF9160_PCA10090` because this module and `bsdlib` do run on the Thingy:91 as well. Any thoughts?